helm/cluster-azure/values.schema.json

```diff
@@ -197,8 +197,29 @@
                     "title": "Identity",
                     "type": "object"
                 },
+                "image": {
+                    "properties": {
+                        "gallery": {
+                            "default": "gsCAPITest1-5cb24dcf-a2d0-4aba-820f-b52ca78f96e6",
+                            "title": "Name of the Community gallery hosting the image",
+                            "type": "string"
+                        },
+                        "name": {
+                            "default": "{{ printf \"capi-flatcar-stable-%s-gen2\" .Values.internal.kubernetesVersion }}",
+                            "title": "Name of the image definition this is parsed as a `gotpl`",
```

Comment:
Again I would make the title shorter, e. g. `Name template`, and add a description.